--- lxr-0.9.4/templates/lxr.conf.orig	2005-04-30 23:30:07.000000000 +0200
+++ lxr-0.9.4/templates/lxr.conf	2006-02-21 17:17:48.000000000 +0100
@@ -8,11 +8,11 @@
 
 		# Path to glimpse executables.
 		# Define this OR the swish-e variable depending which search engine you want to use.
-		'glimpsebin'   => '/info/lxr/bin/glimpse',
-		'glimpseindex' => '/info/lxr/bin/glimpseindex',
+		#'glimpsebin'   => '/usr/bin/glimpse',
+		#'glimpseindex' => '/usr/bin/glimpseindex',
 
 		# Location of SWISH-E binary
-		'swishbin' => '/usr/local/bin/swish-e',
+		'swishbin' => '/usr/bin/swish-e',
 
 		# Path to Exuberant Ctags executable
 		'ectagsbin' => '/usr/bin/ctags',
@@ -21,10 +21,10 @@
 		'tmpdir' => '/tmp',
 
 		# Location of the Generic.pm config file
-		'genericconf' => '/path/to/lib/LXR/Lang/generic.conf',
+		'genericconf' => 'PERLVENDOR/LXR/Lang/generic.conf',
 
 		# Location of the ectags.conf file
-		'ectagsconf' => '/path/to/lib/LXR/Lang/ectags.conf',
+		'ectagsconf' => 'PERLVENDOR/LXR/Lang/ectags.conf',
  		
  		# Paths for CVS module
   		'cvspath' => '/bin:/usr/local/bin:/usr/bin:/usr/sbin',
@@ -36,7 +36,7 @@
 		# Configuration for http://192.168.1.3/lxr.
 		# baseurl is used to select configuration block.
 
-		'baseurl' => 'http://192.168.1.3/lxr',    # Put your URL here
+		'baseurl' => 'http://localhost/lxr',    # Put your URL here
 		     # baseurl_aliases allows other URLs to be used to reach the site
 		     # comment this out if you do not want any aliases
 		'baseurl_aliases' => [ 'http://localhost/lxr', 'http://mydomain/lxr' ],
@@ -51,7 +51,7 @@
 				# This can come from a file, a function or be explicitly
 				# ennumerated.
 				# From a file:
-				'range' => [ readfile('src/versions') ],
+				'range' => [ qw(linux) ],
 
 				# Explicitly:
 				# 'range' => [qw(v1 v2 v3.1 v4 experimental)],
@@ -64,13 +64,14 @@
 				#			}, # deferred function call.
 
 				# The default version to display
-				'default' => '1.0.6'
+				'default' => 'linux'
 			},
 
 			# Define typed variable "a".  First value is default.
 			'a' => {
 				'name'  => 'Architecture',
-				'range' => [qw(i386 alpha arm m68k mips ppc sparc sparc64)]
+				'range' => [qw(alpha arm arm26 cris frv h8300 i386 ia64 m32r m68k m68knommu mips parisc powerpc ppc s390 sh sh64 sparc sparc64 um v850 x86_64 xtensa)],
+				'default' => 'i386'
 			},
 		},
 
@@ -101,7 +101,7 @@
 		#  indexed-src/version2/...
 		# The names of the version directories must match the values for the Version
 		# variable above.
-		'sourceroot' => '/home/malcolm/indexed-src',
+		'sourceroot' => '/usr/src/',
 
 		# Alternatively, this can specify a CVS repository by setting the value to "cvs:"
 		# followed by the path to the repository.  Note this must be file accessible - remote
@@ -109,7 +110,7 @@
 		#     'sourceroot'	=> 'cvs:/hom/karsk/a/CVSROOT/linux',
 
 		# The name to display for this source tree
-		'sourcerootname' => 'Example',
+		'sourcerootname' => 'linux',
 
 		# The DBI identifier for the database to use
 		# For mysql, the format is dbi:mysql:dbname=<name>
@@ -119,8 +119,8 @@
 
 		# If you need to specify the username or password for the database connection,
 		# uncomment the following two lines
-		#  'dbpass'		=> 'foo',
-		#  'dbuser'		=> 'lxr',
+		  'dbpass'		=> 'lxr',
+		  'dbuser'		=> 'lxr',
 
 		# If you need multiple lxr configurations in one database, set different table
 		# prefixes for them.
@@ -127,10 +127,10 @@
 		#  'dbprefix' => 'lxr_',
 
 		# For using glimpse, the directory to store the .glimpse files in is required
-		'glimpsedir' => '/path/to/glimpse/databases',
+		'glimpsedir' => '/var/lib/lxr/glimpse',
 
 		# Location of swish-e index database files if using swish-e
-		'swishdir' => '/a/directory/here/',
+		'swishdir' => '/var/lib/lxr/swish',
 
 		# where to look for include files inside the sourcetree.  This is used to hyperlink
 		# to included files.
