--- lxr-0.9.9/templates/lxr.conf.orig	2011-04-12 11:17:51.000000000 +0200
+++ lxr-0.9.9/templates/lxr.conf	2011-04-12 11:29:12.173904238 +0200
@@ -17,15 +17,15 @@
 
 		# Define this OR the swish-e variable depending which search engine you want to use.
 		# Path to glimpse executables.
-	, 'glimpsebin'   => '/usr/local/bin/glimpse'
-	, 'glimpseindex' => '/usr/local/bin/glimpseindex'
+	, 'glimpsebin'   => '/usr/bin/glimpse'
+	, 'glimpseindex' => '/usr/bin/glimpseindex'
 		# Location of SWISH-E binary
-	, 'swishbin' => '/usr/local/bin/swish-e'
+	, 'swishbin' => '/usr/bin/swish-e'
 
 		# Path to Exuberant Ctags executable
 	, 'ectagsbin' => '/usr/bin/ctags'
 		# Location of the ectags.conf file
-	, 'ectagsconf' => '/path/to/lib/LXR/Lang/ectags.conf'
+	, 'ectagsconf' => '@@PERLVENDOR@@/LXR/Lang/ectags.conf'
 
 		# Paths for CVS module
 	, 'cvspath' => '/bin:/usr/local/bin:/usr/bin:/usr/sbin'
@@ -67,7 +67,7 @@
 		'(bitmap|bmp|gif|icon|jp2|jpg|jpeg|pjpg|pjpeg|png|tif|tiff|xbm|xpm)'
 
 		# Location of the Generic.pm config file
-	, 'genericconf' => '/path/to/lib/LXR/Lang/generic.conf'
+	, 'genericconf' => '@@PERLVENDOR@@/LXR/Lang/generic.conf'
 
 		# How to map files to languages
 		# Note that the string for the key and the first entry in the
@@ -163,7 +163,7 @@
 
 		# Configuration for http://192.168.1.3/lxr.
 		# baseurl is used to select configuration block.
-	  'baseurl' => 'http://192.168.1.3/lxr'    # Put your URL here
+	  'baseurl' => 'http://localhost/lxr'    # Put your URL here
 		     # baseurl_aliases allows other URLs to be used to reach the site
 		     # comment this out if you do not want any aliases
 	, 'baseurl_aliases' =>
@@ -197,7 +197,7 @@
 		#  indexed-src/version2/...
 		# The names of the version directories must match the values for the Version
 		# variable (see the version selection subsection).
-	, 'sourceroot' => '/home/malcolm/indexed-src'
+	, 'sourceroot' => '/usr/src'
 
 		# Alternatively, this can specify a CVS repository by setting the value to "cvs:"
 		# followed by the path to the repository.  Note this must be file accessible - remote
@@ -232,7 +232,7 @@
 		#		annotations are switched on, too!
 		#
 	, 'sourceparams' =>
-			{ 'cachepath'	=> '/a/path/here'
+			{ 'cachepath'	=> '/var/tmp'
 			, 'do_annotate'	=> 1,
 			, 'do_blame'	=> 1
 			}
@@ -351,9 +351,9 @@
 #	, 'dbprefix' => 'lxr_'
 
 		# For using glimpse, the directory to store the .glimpse files in is required
-	, 'glimpsedir' => '/path/to/glimpse/databases'
+	, 'glimpsedir' => '/var/lib/lxr/glimpse'
 
 		# Location of swish-e index database files if using swish-e
-	, 'swishdir' => '/a/directory/here/'
+	, 'swishdir' => '/var/lib/lxr/swish'
 	}
 )
