diff -ur lxr-0.9.4/initdb-mysql lxr-0.9.4-mysql5/initdb-mysql
--- lxr-0.9.4/initdb-mysql	2004-07-20 17:31:24.000000000 +0200
+++ lxr-0.9.4-mysql5/initdb-mysql	2006-02-21 19:20:33.000000000 +0100
@@ -34,8 +34,8 @@
 
 create table lxr_releases 
         (fileid         int not null references lxr_files,
-        release         char(255) binary not null,
-        primary key     (fileid,release)
+        `release`       char(255) binary not null,
+        primary key     (fileid,`release`)
 );
 
 create table lxr_useage
diff -ur lxr-0.9.4/lib/LXR/Index/Mysql.pm lxr-0.9.4-mysql5/lib/LXR/Index/Mysql.pm
--- lxr-0.9.4/lib/LXR/Index/Mysql.pm	2004-10-18 22:45:37.000000000 +0200
+++ lxr-0.9.4-mysql5/lib/LXR/Index/Mysql.pm	2006-02-21 19:20:13.000000000 +0100
@@ -71,16 +71,16 @@
 		  . "where s.symid = i.symid and i.fileid = f.fileid "
 		  . "and f.fileid = r.fileid "
 		  . "and i.langid = d.langid and i.type = d.declid "
-		  . "and  s.symname = ? and  r.release = ?");
+		  . "and  s.symname = ? and  r.`release` = ?");
 	$self->{indexes_insert} =
 	  $self->{dbh}->prepare(
 		"insert into ${prefix}indexes (symid, fileid, line, langid, type, relsym) values (?, ?, ?, ?, ?, ?)"
 	  );
 
 	$self->{releases_select} =
-	  $self->{dbh}->prepare("select * from ${prefix}releases where fileid = ? and  release = ?");
+	  $self->{dbh}->prepare("select * from ${prefix}releases where fileid = ? and  `release` = ?");
 	$self->{releases_insert} =
-	  $self->{dbh}->prepare("insert into ${prefix}releases (fileid, release) values (?, ?)");
+	  $self->{dbh}->prepare("insert into ${prefix}releases (fileid, `release`) values (?, ?)");
 
 	$self->{status_get} =
 	  $self->{dbh}->prepare("select status from ${prefix}status where fileid = ?");
@@ -102,7 +102,7 @@
 		  . "where s.symid = u.symid "
 		  . "and f.fileid = u.fileid "
 		  . "and u.fileid = r.fileid "
-		  . "and s.symname = ? and  r.release = ? "
+		  . "and s.symname = ? and  r.`release` = ? "
 		  . "order by f.filename");
 	$self->{decl_select} =
 	  $self->{dbh}->prepare(
@@ -115,24 +115,24 @@
 	  $self->{dbh}->prepare("delete from ${prefix}indexes "
 		  . "using ${prefix}indexes i, ${prefix}releases r "
 		  . "where i.fileid = r.fileid "
-		  . "and r.release = ?");
+		  . "and r.`release` = ?");
 	$self->{delete_useage} =
 	  $self->{dbh}->prepare("delete from ${prefix}useage "
 		  . "using ${prefix}useage u, ${prefix}releases r "
 		  . "where u.fileid = r.fileid "
-		  . "and r.release = ?");
+		  . "and r.`release` = ?");
 	$self->{delete_status} =
 	  $self->{dbh}->prepare("delete from ${prefix}status "
 		  . "using ${prefix}status s, ${prefix}releases r "
 		  . "where s.fileid = r.fileid "
-		  . "and r.release = ?");
+		  . "and r.`release` = ?");
 	$self->{delete_releases} =
-	  $self->{dbh}->prepare("delete from ${prefix}releases " . "where release = ?");
+	  $self->{dbh}->prepare("delete from ${prefix}releases " . "where `release` = ?");
 	$self->{delete_files} =
 	  $self->{dbh}->prepare("delete from ${prefix}files "
 		  . "using ${prefix}files f, ${prefix}releases r "
 		  . "where f.fileid = r.fileid "
-		  . "and r.release = ?");
+		  . "and r.`release` = ?");
 
 	return $self;
 }
